src/main/resources/resources/cohortcharacterizations/sql/distributionWithCriteria.sql

```diff
@@ -9,49 +10,51 @@ WITH qualified_events AS (
 )
 select
   v.person_id as person_id,
-  count(*) as value_as_int
+  @valueExpression as value_as_number
 into #events_count
 from ( @groupQuery ) v
-group by v.person_id;
+{@aggregateQuery != ""} ? {join qualified_events E on E.person_id = v.person_id @aggregateQuery}
+{@useAggregatedValue} ? {group by v.person_id}
```

Comment:
Looking at this query, I think there may be some confusion around how this feature should work. As @chrisknoll mentioned in https://github.com/OHDSI/WebAPI/issues/495#issuecomment-431905691: 

> I think the distribution type of criteria is going to need to be more sophisticated: getting a distribution means that you're projecting some value per person (visit counts, distinct drug concepts, total cost, visit duration, etc) that comes from a set of records. So the criteria can be leveraged to identify those records we wish to calculate some value from, but what we'd also need is a way to specify what value we are getting from the data that we want the distribution of.

In this code, it looks like the `@aggregateQuery` will use a subquery like:

`join measurement m on m.person_id = v.person_id and m.measurement_date between E.start_date and E.end_date`

based on the values I see here: https://github.com/OHDSI/WebAPI/pull/1449/files?file-filters%5B%5D=.sql#diff-0301f6fb94f1e63492a55537fc105e72R18. 

I think logically what your aiming to do is right: we want to get a distribution based on the events that are defined in the custom feature. I think though that grouping only on the `person_id` is problematic. Take for instance the measurement aggregate query: in this case, grouping only on person_id is problematic because you are not also considering the different measurement concepts - your only aggregating the value which doesn't work since values are specific to some measurement concept.